backend/app/Usecases/Order/InitOrderAction.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace App\Usecases\Order;
+
+use App\Constants\MessageConst;
+use App\Models\Order;
+use App\Models\Tenant;
+use App\Usecases\Order\Exceptions\ActiveOrderAlreadyExistException;
+use App\Usecases\Order\Exceptions\CreateTokenFailedException;
+use App\Usecases\Order\Exceptions\TableIDInvalidException;
+use Firebase\JWT\JWT;
+
+class InitOrderAction
+{
+    public function __invoke(Tenant $tenant, int $tableNumber): array
+    {
+        if ($tableNumber > $tenant->table_count || $tableNumber <= 0) {
+            throw new TableIDInvalidException(MessageConst::TABLE_NUMBER_INVALID);
+        }
+
+        $isExist = Order::where('tenant_id', $tenant->id)
+            ->where('table_number', $tableNumber)
+            ->where('status', Order::STATUS_OPEN)
+            ->first();
```

Comment:
存在判定であれば、existsのほうがいいかもです。

```suggestion
            ->exists();
```

https://readouble.com/laravel/11.x/ja/queries.html

>レコード存在の判定

Reply:
first()に戻ってるかもです。。。

Reply:
↑これでいいです。ロジックが変わってました